README.md

```diff
@@ -1,5 +1,8 @@
 # Azure landing zones Terraform module
 
+[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/azure/terraform-azurerm-caf-enterprise-scale.svg)](http://isitmaintained.com/project/azure/terraform-azurerm-caf-enterprise-scale "Average time to resolve an issue")
```

Comment:
Can we put these alongside the others rather than above?

Just need to be moved below the existing badges, with no blank line between.